### PR TITLE
[Settings] Adjust Preferences

### PR DESCRIPTION
**Pull request by Florian14**
#268

### CODE REVIEW

File: vector/src/main/res/xml/vector_settings_preferences.xml [hunk 1]
```diff
@@ -82,7 +82,8 @@
             android:defaultValue="true"
```
Comment (giomfo) [inline vector/src/main/res/xml/vector_settings_preferences.xml:82]: I forgot to ask you to turn this pref off, please set false by default
Please consider the default in VectorPreferences.kt too

File: vector/src/main/java/im/vector/app/features/settings/VectorPreferences.kt [hunk 2]
```diff
@@ -415,7 +415,7 @@ class VectorPreferences @Inject constructor(private val context: Context) {
      * @return true true if the avatar and display name events should be shown in the messages list.
      */
     fun showAvatarDisplayNameChangeMessages(): Boolean {
-        return defaultPrefs.getBoolean(SETTINGS_SHOW_AVATAR_DISPLAY_NAME_CHANGES_MESSAGES_KEY, true)
+        return defaultPrefs.getBoolean(SETTINGS_SHOW_AVATAR_DISPLAY_NAME_CHANGES_MESSAGES_KEY, false)
```
Comment (giomfo) [inline vector/src/main/java/im/vector/app/features/settings/VectorPreferences.kt:418]: I unsintalled the application and install a new one by using your branch, for unknown reasons these 2 preferences were enabled after a new connexion. This looks like the default values is overridden somewhere else
Can you please test on your side?

Comment (giomfo) [reply vector/src/main/java/im/vector/app/features/settings/VectorPreferences.kt:418]: I think I found where (see https://github.com/tchapgouv/tchap-android-v2/pull/277/files#diff-f71c85b34c236d34da29a61fa263f190f0e5b7664a531e84741cbc90c003d234L109 and https://github.com/tchapgouv/tchap-android-v2/pull/277/files#diff-f71c85b34c236d34da29a61fa263f190f0e5b7664a531e84741cbc90c003d234L115)

**Comment by Florian14** [comment]
rebased

**Review by giomfo — COMMENTED** [review]
@Florian14 you remove dup in vector_settings_pref (SETTINGS_SHOW_JOIN_LEAVE_MESSAGES_KEY, SETTINGS_SHOW_AVATAR_DISPLAY_NAME_CHANGES_MESSAGES_KEY) OK
but you didn't update the defaultValue of the kept settings (SETTINGS_SHOW_JOIN_LEAVE_MESSAGES_KEY, SETTINGS_SHOW_AVATAR_DISPLAY_NAME_CHANGES_MESSAGES_KEY

**Review by giomfo — APPROVED** [review]
Tests OK

**Pull request merged**